tfmkt/spiders/players.py

```diff
@@ -73,6 +73,8 @@ def parse_details(self, response, base):
     attributes['contract_expires'] = self.safe_strip(response.xpath("//span[text()='Contract expires:']/following::span[1]/text()").get())
     attributes['day_of_last_contract_extension'] = response.xpath("//span[text()='Date of last contract extension:']/following::span[1]/text()").get()
     attributes['outfitter'] = response.xpath("//span[text()='Outfitter:']/following::span[1]/text()").get()
+    attributes['current_market_value'] = response.xpath("//div[@class='marktwertentwicklung']//div[@class='zeile-oben']//div[@class='right-td']//a/text()").get().strip()
```

Comment:
This looks great @DonFloriano27!
Just one tiny detail: would you mind using the `self.safe_strip` method to do the stripping instead of calling `.strip()` on the string? This will make a little bit more robust and it wont' fail in case if the scraped value is a null.